### PR TITLE
BaSP-MR-205: Changed the way to obtain subscription data based on the new populate

### PR DESCRIPTION
**Pull request by josedaniele**
The way data is obtained from the subscriptions table in the member has been changed based on the new populate in subscriptions, while maintaining the same functionality as before.

### CODE REVIEW

**Review by IgnacioGalceran — APPROVED** [review]
Everything works

File: src/Components/Members/SubscriptionsMember/index.jsx [hunk 1]
```diff
@@ -25,33 +23,13 @@ const SubscriptionsMember = () => {
...
-            day: sub.classes.day,
-            time: sub.classes.time
-          });
-        }
-      });
-    });
-    return arraySubs;
-  };
+    setSubscription(filterSubscription);
+    console.log(filterSubscription);
```
Comment (jramire5) [inline src/Components/Members/SubscriptionsMember/index.jsx]: Please take this out!

File: src/Components/Members/SubscriptionsMember/index.jsx [hunk 2]
```diff
@@ -60,6 +38,7 @@ const SubscriptionsMember = () => {
 
   const handleDeleteButton = (id) => {
     setShowConfirmModal(true);
+    console.log(id);
```
Comment (jramire5) [inline src/Components/Members/SubscriptionsMember/index.jsx]: and this clg

File: src/Components/Members/SubscriptionsMember/index.jsx [hunk 3]
```diff
@@ -25,33 +23,13 @@ const SubscriptionsMember = () => {
 
   useEffect(() => {
     dispatch(getSubscriptions);
-    getActivities(dispatch);
   }, [dispatch]);
```
Comment (jramire5) [inline src/Components/Members/SubscriptionsMember/index.jsx]: the dependency array doesn't need dispatch

**Review by jramire5 — APPROVED** [review]
Good Job!

**Pull request merged**